README.md

```diff
@@ -1,23 +1,31 @@
-# care-for-all
+# Care For All
+
 python -m venv env #create a virtul environment
 env\Scripts\activate # activate the enviroment
 pip install -r requirements.txt
-# NB MAKE SURE YOUR VIRTUAL ENVIRONMENT IS ACTIVE AT ALL TIME  
 
-# if first time
-# Back-End
+## NB: MAKE SURE YOUR VIRTUAL ENVIRONMENT IS ACTIVE AT ALL TIME BEFORE RUNNING THE BELOW
+
+## First Time Run (Installing dependencies)
+
+### (1) Back-End
+
 pip install django
+pip install djangorestframework
+pip install django-cors-headers
+
+## (2) Run Front-End
+
+npm install -g expo-cli
+
+## Run Back-End
 
-# To run the Back-end
 cd backend(where theres the manage.py)
 python manage.py makemigrations
 python manage.py migrate
 python manage.py runserver
 
+## Run Front-End
 
-# Front-End
-npm install -g expo-cli
-
-# To run the Front-end
 cd front-end
-expo start
+npm run web
```

Comment:
No need to specify, the expo start will give option on where you wanna run it
Or just run npm start

Reply:
Ayt. Removed this change.